src/components/DropdownButton/DropdownButton.module.css

```diff
@@ -25,8 +25,7 @@
   align-items: center;
 
   cursor: pointer;
-
-  background-color: var(--eds-theme-color-background-neutral-default);
+  @apply bg-background-neutral-default-default;
```

Comment:
Hi @schakraborty22 sorry for getting back to this late but we want to find a way to compile the TW styles to use `bg-background-neutral-default` and not `bg-background-neutral-default-default`. This is occurring due to `css-variables-nested.json` being 
```css
"background": {
  "neutral": {
    "default": {
      "hover": "var(--eds-theme-color-background-neutral-default-hover)",
      "default": "var(--eds-theme-color-background-neutral-default)"
    }
  }
}
```
Previously 
```css
```css
"background": {
  "neutral": {
    "default": {
      "hover": "var(--eds-theme-color-background-neutral-default-hover)",
      "@": "var(--eds-theme-color-background-neutral-default)"
    }
  }
}
```
would become `bg-background-neutral-default-@" which we're also trying to avoid.